test/e2e/fixtures/send-edit-v2/state.json

```diff
@@ -130,6 +130,19 @@
       "useNonceField": false,
       "usePhishDetect": true
     },
+    "MetaMetricsController": {
+      "fragments": {
+        "transaction-added-4046084157914634": {
```

Comment:
[REQ] I think we should be doubly safe and make sure that fragments exist in userland before calling them. Scenario: User submits a transaction with low max fee (doesn't care how long it takes) before the new version goes out... they don't have a fragment for transaction-added. This would cause a failure.

There's a function on the transaction controller called createTransactionEventFragment that I had intended the UI hook to call to ensure the existence of the fragment. If you call this method with the event as TRANSACTION_EVENTS.APPROVED constant it'll create the fragment if it doesn't exist. Its a safety net. Just ensure we only call this method the first time the hook is executed to avoid over calling the method.